modules/io/common/src/main/java/org/locationtech/jts/io/geojson/GeoJsonWriter.java

```diff
@@ -88,6 +89,16 @@ public void setEncodeCRS(boolean isEncodeCRS) {
     this.isEncodeCRS  = isEncodeCRS;
   }
   
+  /**
+   * Sets whether the GeoJSON should be output following counter-clockwise orientation aka Right Hand Rule defined in RFC7946
+   * See <a href="https://tools.ietf.org/html/rfc7946#section-3.1.6">RFC 7946 Specification</a> for more context.
+   *
+   * @param enforceCCW true if the GeoJSON should be output following the RFC7946 counter-clockwise orientation aka Right Hand Rule
+   */
+  public void setEnforceCCW(boolean enforceCCW) {
```

Comment:
One final request.  In the interest of brevity, can this be changed to `setForceCCW` and `isForceCCW`?

Reply:
Done and done. Don't hold back on the requests, they have all been on the money and I'm a big fan of things done right 😄👍👍